samples/hello-oboe/src/main/java/com/google/sample/oboe/hellooboe/MainActivity.java

```diff
@@ -49,6 +49,7 @@
     private static final int CHANNEL_COUNT_DEFAULT_OPTION_INDEX = 1;
     private static final int[] BUFFER_SIZE_OPTIONS = {0, 1, 2, 4, 8};
     private static final String[] AUDIO_API_OPTIONS = {"Unspecified", "OpenSL ES", "AAudio"};
+    private static final int SPINNER_DEFAULT_OPTION_INDEX = 0;
```

Comment:
Worth adding a comment here: 

    // Default all other spinners to the first option in the list

Reply:
Added in following commit